### PR TITLE
Add card blocks and wide card modifier

### PR DESCRIPTION
**Pull request by prestonp**

### CODE REVIEW

File: packages/cf-component-card/src/CardControl.js [hunk 1]
```diff
@@ -3,16 +3,19 @@ const {PropTypes} = React;
 
 class CardControl extends React.Component {
   render() {
+    const className = 'cf-card__control' + (this.props.wide ? ' cf-card__control--wide' : '');
```
Comment (jculvey) [inline packages/cf-component-card/src/CardControl.js]: I've found the [the `classnames` library](https://github.com/JedWatson/classnames) library to be useful at handling this particular oft-occurring problem.

Comment (prestonp) [inline packages/cf-component-card/src/CardControl.js]: Hey that's really cool! I think this should be addressed across the entire repo so I opened  issue #49

Comment (jculvey) [inline packages/cf-component-card/src/CardControl.js]: Awesome!

**Comment by wyuenho** [comment]
Why do we need yet another extra card block as opposed to just a div?

**Comment by prestonp** [comment]
I just wanted to make it an easy component. Of course, anything customization could just be defined downstream but we use this pattern a lot for multiple sections within a CardControl

**Comment by wyuenho** [comment]
Don't we already have a bunch of layout components you can use to align the blocks?

**Comment by prestonp** [comment]
The closest thing I saw was .cf-form__fieldset_content from the Form components but would still take some modification. I think this is intuitive and easy to read:

```jsx
<CardControl>
  <CardBlock><button ... /></CardBlock>
  <CardBlock><button ... /></CardBlock>
  <CardBlock><button ... /></CardBlock>
</CardControl>
```

**Pull request merged**